CHAOS-6693 Image registry usage at different scopes (#8810)
harness/developer-hub: docs/chaos-engineering/use-harness-ce/image-registry.md

When the image you need to use for your chaos experiment is private, and the chaos experiments are required to be run for internal consumption, you can configure image registry as **private** and provide custom values to it.
This way, you will have better control, and security when working with private images.

> Depending on whether you use DDCR (Delegate Driven Chaos Runner) or dedicated chaos infrastructure, image registry settings is configured from **account**/**organization**/**project** settings or from the UI, respectively.

Reviewer: ```suggestion
Depending on whether you use DDCR (Delegate Driven Chaos Runner) or dedicated chaos infrastructure, image registry settings is configured from **account**/**organization**/**project**/**infrastructure** settings or from the UI, respectively.
```

<Tabs>
<TabItem value = "Harness Delegate / DDCR">

> ### Configure Image Registry from Account/Organization/Project settings

Reviewer: ```suggestion
### Configure Image Registry from Account/Organization/Project/Infrastructure settings
```

> With appropriate permissions, you can configure image registry from the **account** or **organization** or **project** settings.

Reviewer: ```suggestion
With appropriate permissions, you can configure image registry from the **account** or **organization** or **project** settings or **infrastructure**.
```

This approach provides flexibility and ensures you can manage image registry settings at different levels.

In this example, you will learn how to configure image registry from **Account Settings**.

1. Go to **Account Settings** -> **Image Registry (For Chaos)**.

    ![account settings](./static/image-registry/account-level.png)

2. Provide registry details:
  a. Specify the server and account name. 
  b. Choose the image registry type. 
  c. Enable **Allow Overrides** option if you want to allow changes to image registry settings at lower levels, such as Organization, Project, or Infrastructure.

>   For example, currently, you are in the **Account** scope. Allowing overrides will allow you to make changes to the image registry settings at the **Organization**, **Project**, and infratructure levels.

Reviewer: ```suggestion
  For example, currently, you are in the **Account** scope. Allowing overrides will allow you to make changes to the image registry settings at the **Organization**, **Project**, and **infrastructure** levels.
```

    ![override settings](./static/image-registry/override.png)

3. Use Custom Images (Optional):

  a. Enable **Use custom images** if you want to provide custom images for the specified fields.
  b. Add your custom images in the fields shown in the screenshot.

    ![custom image settings](./static/image-registry/custom-img.png)

:::tip

> If you enable the **Allow Overrides** option, you can configure the image registry at the infrastructure level:

Reviewer: ```suggestion
If you enable the **Allow Overrides** option, you can configure the image registry for infrastructures that are Supported by a Harness Delegate:
```
